onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_support_checker.cc

```diff
@@ -1639,6 +1654,62 @@ class EluOpSupportChecker : public BaseOpSupportChecker {
 
 #pragma endregion
 
+#pragma region op_slice
+
+class SliceOpSupportChecker : public BaseOpSupportChecker {
+ private:
+  int32_t GetMinSupportedNNAPIFeatureLevel(const Node& /* node */, const OpSupportCheckParams& /* params */) const override {
```

Comment:
nit: there were a bunch of other changes to break similar lines, but this line is a bit long